php_yaf.h

```diff
@@ -122,6 +122,10 @@ extern ZEND_DECLARE_MODULE_GLOBALS(yaf);
 
 #endif
 
+#if PHP_VERSION_ID >= 70400
+typedef unsigned long ulong;
```

Comment:
This doesn't look right wrt. portability. E.g. `unsigned long` is a 32bit value on LLP64, but a 64bit value on LP64. It seems to me that all usages of `ulong` in this extension should actually be `zend_ulong`, and in my opinion should be replace inline instead of providing a potentially misleading typedef. Thanks.

Reply:
hmm, I missed this part, fixed . thanks